app/views/juke/index.html.erb

```diff
@@ -38,7 +38,35 @@
   </h2>
 
 </section>
+<div>
+<h1 id="time"></h1>
+<script type='text/javascript'>
+	function el(tid) {return document.getElementById(tid);}
+	function serverTime(s){
+		el("time").innerHTML= 	"Time is " + s.hour +
+							":" + s.minute +
+							":"+s.second;
+	}
+</script>
+</div>
+
+<<<<<<< HEAD
```

Comment:
merge conflict - you need to pull latest master, merge with branch and resolve conflict